src/DynamoCore/Graph/Annotations/AnnotationModel.cs

```diff
@@ -70,13 +65,16 @@ public class AnnotationModel : ModelBase
         /// </summary>
         public override double Width
         {
-            get => width;
+            get
+            {
+                return width;
+            }
             set
             {
                 if (width == value) return;
 
                 width = value;
-                RaisePropertyChanged(nameof(Width));
+                RaisePropertyChanged("Width");
```

Comment:
I know you are reverting the whole change but we should just change this back to use nameof.

Reply:
there will be follow up PR to re-implement this, we can handle this case there.
fyi @ivaylo-matov